evalml/pipelines/components/transformers/encoders/target_encoder.py

```diff
@@ -56,7 +55,6 @@ def __init__(self,
         category_encode = import_or_raise('category_encoders', error_msg='category_encoders not installed. Please install using `pip install category_encoders`')
```

Comment:
Same here, since `random_seed` is missing, can we add it to the docstring?

Reply:
Done!